pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -689,7 +689,17 @@ func (r *RepoSyncReconciler) populateContainerEnvs(ctx context.Context, rs *v1be
 	case v1beta1.OciSource:
 		result[reconcilermanager.OciSync] = ociSyncEnvs(rs.Spec.Oci.Image, rs.Spec.Oci.Auth, v1beta1.GetPeriodSecs(rs.Spec.Oci.Period))
 	case v1beta1.HelmSource:
-		result[reconcilermanager.HelmSync] = helmSyncEnvs(rs.Spec.Helm)
+		result[reconcilermanager.HelmSync] = helmSyncEnvs(helmOptions{
+			repo:        rs.Spec.Helm.Repo,
+			chart:       rs.Spec.Helm.Chart,
+			version:     rs.Spec.Helm.Version,
```

Comment:
What about making `helmSyncEnvs` takes HelmBase and a namespace as parameters, so that you don't need to build the helmOptions.

Reply:
Sounds good. Done.